<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegionCurvatureEdgeCorrection</class>
 <widget class="QWidget" name="RegionCurvatureEdgeCorrection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Find Edge Pixels</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Seed Point</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="seed_point_label">
           <property name="text">
            <string>x | y | z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="set_seed_point_button">
           <property name="text">
            <string>Set Seed Point</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Tolerance</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QDoubleSpinBox" name="tolerance_spinbox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="singleStep">
            <double>0.000010000000000</double>
           </property>
           <property name="value">
            <double>0.000970000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Edge Interpolation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Pixels to leave</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QSpinBox" name="count_of_pixels_to_leave_spinbox">
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Node Pixels</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QSpinBox" name="count_of_node_pixels_spinbox">
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>PIxels to generate</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QSpinBox" name="count_of_pixels_to_generate_spinbox">
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="perform_button">
     <property name="text">
      <string>Perform</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
